wallet/src/de/schildbach/wallet/ui/ExchangeRatesFragment.java

```diff
@@ -390,19 +399,31 @@ void setExchangeRates(List<de.schildbach.wallet.rates.ExchangeRate> exchangeRate
         }
     }
 
+    private Integer getFlagFromCurrencyCode(String currencyCode) {
+        if (currencyCode.equalsIgnoreCase("try")) return R.drawable.turk;    //we can't use 'try' as the recource name for turkish flag as try is a reserved keyword in java. A workaround is to use rename the turkish flag resource
```

Comment:
We should avoid such weird exceptions. I would suggest to add a prefix to each resource name (e.g. "currency_code_try", or something similar). Additionally, names with such prefix would be more self-descriptive.